public/app/features/alerting/unified/components/rule-editor/notificaton-preview/NotificationRoute.tsx

```diff
@@ -92,6 +95,7 @@ function NotificationRouteHeader({
           onClose={() => setShowDetails(false)}
           route={route}
           receiver={receiver}
+          receiverName={receiverName}
```

Comment:
How about renaming it to `unknownReceiverName` or something like that and pass it only if `receiver` is null/undefined?
It's a bit strange that we need to pass `receiver` and `receiverName`, even though `receiver` contains the `name` field. I'm wondering how to make it more clear

Reply:
`route.receiver` being a receiver's name doesn't help either 😅